package.json

```diff
@@ -57,7 +59,18 @@
     "punycode": "^1.4.1",
     "rimraf": "^3.0.2",
     "semver": "^5.7.1",
-    "sinon": "^7.3.1"
+    "sinon": "^7.3.1",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
+    "@babel/plugin-proposal-optional-chaining": "^7.2.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-plugin-minify-dead-code-elimination": "^0.5.0",
+    "babel-plugin-transform-inline-environment-variables": "^0.4.3",
+    "read-pkg-up": "^4.0.",
+    "resolve": "^1.22.1"
```

Comment:
These are dependencies of the CLI when running in development mode. They are required by the cli for transformations. 

This lists them explicitly as dependencies that need to made available globally.

They are needed to be made available globally due to the parcelBableRegister setting it's cwd to the project root: https://github.com/parcel-bundler/parcel/blob/alsh/rationalizing-phantom-dependencies/packages/dev/babel-register/index.js#L5

Reply:
I had thought that Babel resolves plugins declared in presets relative to that preset, and they are declared over there: https://github.com/parcel-bundler/parcel/tree/v2/packages/dev/babel-preset
Otherwise you'd have this problem with every Babel preset from npm as well?

Reply:
You're right, I spent some time trying to understand where this was coming from and it turns out we use `@parcel/babel-preset` from the root level in the `babel.config.json`. So when I used strict workspace package isolation, babel would complain.

Turns out I just needed to add `@parcel/babel-preset` to the workspace `package.json`.

Removed the unused extras